### DISCUSSION TITLE
[Feature] display path of images in Libraries

### DISCUSSION THREAD

**Discussion by ghost** [immich-app/immich#4159] [Feature Request]
### The feature
show the image path in the detail view.
it will be very helpful for managing files on disk

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by ghost** [comment]
it would be even better to atomically create album according to the folder names.

**Comment by davidrudlstorfer** [comment]
Just wanted to create the same feature request as I've already mentioned these two points in a discussion on Discord 😅

In my opinion the optimal way would be to just show the image path in the detail view.

Regarding the creation of albums, I do not know how complicated it will be but as you've mentioned the albums could easily reproduce the hierarchical structure present in the library.

**Comment by ghost** [comment]


**Reply by jrasm91** [reply]
Yes?

**Comment by nodis** [comment]
Yes, I support this suggestion
